contracts/governance/comp/GovernorVault.sol

```diff
@@ -0,0 +1,52 @@
+pragma solidity ^0.5.17;
+
+import "../../openzeppelin/Ownable.sol";
+import "../../interfaces/IERC20.sol";
+
+contract GovernorVault is Ownable {
+
+    string public constant ERROR_INVALID_ADDRESS = "Invalid address";
+
+    IERC20 public token;
+
+    event Deposited(address indexed sender, uint amount);
+    event TokensTransferred(address indexed receiver, uint amount);
+    event RbtcTransferred(address indexed receiver, uint amount);
+
+    constructor(address _token) public {
+        require(_token != address(0), ERROR_INVALID_ADDRESS);
+
+        token = IERC20(_token);
+    }
+
+    /**
+     * @notice transfers tokens
+     * @param _receiver the receiver of tokens
+     * @param _amount the to be transferred
+     */
+    function transferTokens(address _receiver, uint _amount) public onlyOwner {
+        require(_receiver != address(0), ERROR_INVALID_ADDRESS);
+
+        require(token.transfer(_receiver, _amount), "Transfer failed");
+        emit TokensTransferred(_receiver, _amount);
+    }
```

Comment:
this will work for this purpose. for the actual governance i would prefer to not be limited to a single token contract, but instead just pass the token address to the transferToken function

Reply:
fixed